loleaflet/src/layer/marker/Cursor.ts

```diff
@@ -56,7 +56,11 @@ class Cursor {
 		this.map.getCursorOverlayContainer().appendChild(this.container);
 		this.visible = true;
 		this.domAttached = true;
-		this.onFocusBlur(new FocusEvent('focus'));
+
+		var isIE11 = !!window.MSInputMethodContext && !!(<any>document).documentMode;
+		if (!isIE11)
+			this.onFocusBlur(new FocusEvent('focus'));
```

Comment:
so in IE11 we just don't support hiding cursor when window is inactive?

Reply:
Yes it seems, other than that, everything seems fine.